scripts/technical-debt-metrics.sh

```diff
@@ -162,10 +162,11 @@ then
   else
     printf '%s\n' "${rep}" |
       awk -F '|' -v rep="${rep}" '
-        BEGIN{total=0; weight=0}
+        BEGIN{total=0; weight=0; absWeight=0}
+        {absWeight+=$3+0}
```

Comment:
Is $3 a shell positional argument (or, e.g., some awk local variable)?
In the former case, please
- document the expected argument to the script in the top-level comment,
- give it a default value (as the script uses `set -u` at the top)

In the latter case... perhaps this is fine. Or a comment, for the confused non-awk-experts like me, is helpful: up to you.

Reply:
`$3` is the "field number" that `awk` uses: in this case, `awk` is splitting the input using `|` as a field separator, so it reads a line, and calls each substring contained between consecutive `|` with an increasing index.  So, `$3` is what happens in
```bash
not here| neither here| yes, here!|not anymore...
```

The input of `awk` is (basically) `report` that has
```bash
|Current number|Change|Type|
```
as header, so `$3` really means whatever happens in `Change`.

I'll add a "general" comment to what the fields represent at the `awk` call, rather than here where the variable is used.

Reply:
Ah, makes sense! Thanks a lot for explaining!